core/frontend/src/store/beacon.ts

```diff
@@ -33,7 +33,7 @@ class BeaconStore extends VuexModule {
 
   nginx_ip_address = ''
 
-  hostname: undefined | string = undefined
+  hostname = ''
```

Comment:
why is this an empty string ? with that there is no way for us to check if the value was proper set/valid or not.

Reply:
How is this different from the other properties?

Reply:
Well for IP consistency it makes sense.. but for valid check empty string is not the proper way.